REGRESSION_TESTS.md

```diff
@@ -0,0 +1,43 @@
+This document assumes you have already been able to build CPROVER on
+your chosen architecture. For details on building CPROVER see
+COMPILING.md.
+
+Regression tests can be run using make or cmake. Your choice here
+should be the same as the compiling of the project itself. (See
+COMPILING.md for details of both make and cmake build processes.)
+
+Note that running all regression tests can be slow when a non-
+release build of CPROVER is used. It is highly recommended to
+compile with release flags.
+
+# MAKE
+
+This can be done by changing to the regression directory and
+simply running make as follows.
+```
+cd regression
+make
+```
+
+# CMAKE
+
+To build with release flags in cmake use
+`-DCMAKE_BUILD_TYPE=Release` as an argument to the cmake command.
```

Comment:
This is unrelated to running the regression tests (it's already mentioned in `COMPILING.md` at any rate).

Reply:
Agree it is redundant. Removed.

Reply:
That's already documented in COMPILING.md?!